<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7SM6G84" failures="0" tests="3" name="Practice.TestNG.ContactTest" time="0.019" errors="1" timestamp="2024-10-17T12:08:14 IST" skipped="0">
  <testcase classname="Practice.TestNG.ContactTest" name="CreateContactTest" time="0.013">
    <error message="Unresolved compilation problems: 
	Syntax error on token &quot;==&quot;, invalid Expression
	Syntax error, insert &quot;)&quot; to complete Expression
	Syntax error, insert &quot;;&quot; to complete BlockStatements
	String literal is not properly closed by a double-quote
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
Syntax error on token "==", invalid Expression
Syntax error, insert ")" to complete Expression
Syntax error, insert ";" to complete BlockStatements
String literal is not properly closed by a double-quote

at Practice.TestNG.ContactTest.CreateContactTest(ContactTest.java:9)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- CreateContactTest -->
  <system-out/>
  <testcase classname="Practice.TestNG.ContactTest" name="modifyContactTest" time="0.003"/>
  <system-out/>
  <testcase classname="Practice.TestNG.ContactTest" name="deleteContactTest" time="0.003"/>
  <system-out/>
</testsuite> <!-- Practice.TestNG.ContactTest -->
